### ISSUE TITLE
Omitting runtime_type/runtime_type_version/script_language attributes on newrelic_synthetic_monitor causes failure when attempting to apply

### ISSUE THREAD

**Issue by arhill05** [newrelic/terraform-provider-newrelic#2232]
- [x] Your New Relic `provider` [configuration](#terraform-configuration-files) (sensitive details redacted)
- [x] A list of [affected resources](#affected-resources) and/or data sources
- [x] The [configuration](#terraform-configuration-files) of the resources and/or data sources related to the bug report (i.e. from the list mentioned above)
- [x] Description of the [current behavior](#actual-behavior) (the bug)
- [x] Description of the [expected behavior](#expected-behavior)
- [x] Any related [log output](#debug-output)

### Terraform Version
1.3.0

### Affected Resource(s)
Please list the resources as a list, for example:
- `newrelic_synthetics_monitor`
- `newrelic_synthetics_script_monitor`

This issue affects multiple resources

### Terraform Configuration
> Please include your `provider` configuration (sensitive details redacted) as well as the configuration of the resources and/or data sources related to the bug report.
```hcl
terraform {
  required_version = ">=1.3.0"
  required_providers {
    newrelic = {
      source  = "newrelic/newrelic"
      version = "3.6.0"
    }
  }
}

provider "newrelic" {
  api_key    = var.api_key
  account_id = var.account_id
  region     = "US"
}

resource "newrelic_synthetics_script_monitor" "canary_synthetic_monitor" {
  account_id = var.account_id
  for_each = {
    for app in local.api_apps : app.monitor_name => app
  }

  location_private {
    guid = data.newrelic_synthetics_private_location.REDACTED.id
  }

  name   = "TF_NRV3_${each.value.monitor_name}"
  period = var.synthetic_monitor_period
  status = coalesce(each.value.is_enabled, true) ? "ENABLED" : "DISABLED"
  type   = "SCRIPT_API"

  tag {
    key    = "managed-by-terraform"
    values = [true]
  }

  script = "some-script"
}

resource "newrelic_synthetics_monitor" "simple_monitor" {
  for_each = {
    for app in local.ui_apps : app.monitor_name => app
  }
  period                                  = "EVERY_10_MINUTES"
  status                                  = "ENABLED"
  type                                    = "BROWSER"
  enable_screenshot_on_failure_and_script = true
  bypass_head_request                     = true
  verify_ssl                              = true

  name = "TF_NRV3_${each.value.monitor_name}"
  uri  = each.value.host_name

  tag {
    key    = "managed-by-terraform"
    values = [true]
  }

  locations_private = [
    data.newrelic_synthetics_private_location.REDACTED.id
  ]
}
```

### Actual Behavior
Running `terraform apply` results in an error: `BAD_REQUEST Runtime values are invalid combination.`

### Expected Behavior
What should have happened?
The monitor should have been created without error

### Steps to Reproduce
1. `terraform apply`

### Debug Output
https://gist.github.com/arhill05/774a211c0d89b728b82f2e72fb4ad6e7

### Important Factoids
- This was working up until at least a couple of days ago.
- The documentation on these specifically calls out setting `runtime_type`, `runtime_type_version`, and `script_language` to an empty string in order to use the legacy runtime, but this doesn't work for either resource. Setting them to the newest versions (see below) works, but we are not ready to move to the newer runtime yet and our scripts will not work with it,

Here's what allows the monitors to be created:
```hcl
  # in newrelic_synthetics_script_monitor resource
  script_language      = "JAVASCRIPT"
  runtime_type         = "NODE_API"
  runtime_type_version = "16.10"
  
  # in newrelic_synthetics_monitor resource with type of "BROWSER"
  script_language      = "JAVASCRIPT"
  runtime_type         = "CHROME_BROWSER"
  runtime_type_version = "100"
```

### References
Are there any other GitHub issues (open or closed) or Pull Requests that should be linked here? For example:
- GH-2228

**Comment by leslie-alldridge** [comment]
Also ran into this :(

**Comment by Scarsrcool10** [comment]
Also encountered this issue.

**Comment by sanderblue** [comment]
A fix for this went out in v3.14.0. Please let us know if you continue to see this problem 🙂
